<launch>
  
  <node pkg="detect_avoid" type="laser_clust" name="laser_clust" /> 
  <node pkg="detect_avoid" type="avoid_fms" name="avoid_fms" />
  
</launch>